frontend/integration-tests/tests/olm/catalog.scenario.ts

```diff
@@ -18,14 +18,20 @@ describe('Installing a service from a Catalog Source', () => {
       metadata: {name: 'console-e2e'},
       spec: {
         sourceType: 'grpc',
-        image: 'quay.io/operatorframework/operator-manifests@sha256:ac3140d9f2d2a3cf5446d82048ddf64ad5cd13b31070d1c4b5c689b7272062dc',
+        image: 'quay.io/operator-framework/upstream-community-operators@sha256:5ae28f6de8affdb2a2119565ea950a2a777280b159f03b6ddddf104740571e25',
         displayName: 'Console E2E Operators',
         publisher: 'Red Hat, Inc',
       },
     };
+
     execSync(`echo '${JSON.stringify(catalogSource)}' | kubectl create -n ${testName} -f -`);
-    // FIXME(alecmerdler): Wait until `PackageManifests` are being served from registry pod
-    browser.sleep(30000);
+    await new Promise(resolve => (function checkForPackages() {
```

Comment:
Removed `browser.sleep()` in favor of polling for the gRPC server to start serving `PackageManifests`.

Reply:
> Removed browser.sleep() in favor of polling for the gRPC server to start serving PackageManifests.

Thanks, this looks a lot cleaner